### DISCUSSION TITLE
Specifying custom path to xdg-dbus-proxy

### DISCUSSION THREAD

**Discussion by gabberdancecat** [netblue30/firejail#5945] [Q&A]
Hi all, 

I am running firejail on a system with a very unique package manager (Guix), which installs binaries to `~/.guix-profile/bin` instead of `/usr/bin`. Because of this, the `xdg-dbus-proxy` executable is located at `~/.guix-profile/bin/xdg-dbus-proxy`.

Upon running firejail, I get an error saying:
```Warning: /usr/bin/xdg-dbus-proxy was not found, downgrading dbus-user policy to allow.```
It seems like instead of searching the $PATH for `xdg-dbus-proxy`, firejail searches in `/usr/bin`. 

Is there a way to specify a custom path to the `xdg-dbus-proxy` binary? 
Note: creating a symlink to `/usr/bin` is not an option, because Guix System (similar to NixOS), my current distro, doesn't want me to.

**Comment by rusty-snake** [comment]
With compiling a patch version: yes, without no.

Given that we do not give it anything privileged AFAIK it should be ok to search $PATH for it.

**Reply by gabberdancecat** [reply]
I tried creating a symlink to `/usr/bin` anyways and to my surprise, updating the system configuration didn't remove the symlink, so I think this solution should work just fine. I doubt this will actually break any system functionality. 

On a related note, on Guix System, firejail can't run programs installed through Guix, the package manager, because of the way Guix System handles setuid permissions, but I found a really hacky workaround and made a script for it, so i'll post that on another github discussion after refining it a little.
